src/main/java/org/myrobotlab/swing/ProgramABGui.java

```diff
@@ -194,7 +174,7 @@ public void run() {
           startSessionButton.setText("Start Session");
           startSessionButton.setBackground(Color.RED);
         } else {
-          startSessionButton.setText("Restart Chatbot");
+          startSessionButton.setText("Force reload ( kill csv )");

```

Comment:
End users should have no idea what "kill csv" mean.  Please just label this button  "Restart Chatbot"  (perhaps "Reload Chatbot" .. )   exposing end users to things like needing to "kill csv" is ugly.